docs/agents/sematext-agent/ansible.md

```diff
@@ -0,0 +1,129 @@
+title: Sematext Agent Ansible Installation
+description: Automated configuration management using Ansible playbooks for Sematext infrastructure and services monitoring
+
+Ansible can be used for automating the deployment of [Sematext Agent](https://sematext.com/docs/agents/sematext-agent/) in multiple hosts across your infrastructure. Using the Sematext Agent Ansible role you can easily extend your existing playbooks and [start monitoring](https://sematext.com/docs/monitoring/) thousands of servers within minutes.
+
+## Ansible Introduction
+[Ansible](https://docs.ansible.com) is an open-source automation tool designed for configuration management, application deployment, and task automation. 
+
+Ansible uses the concepts of control and managed nodes. It connects from the [control node](https://docs.ansible.com/ansible/latest/network/getting_started/basic_concepts.html#control-node) where Ansible is installed, to the managed nodes sending commands and instructions to them over SSH protocol.
+
+Ansible utilizes a very simple language, YAML, to define [playbooks](https://docs.ansible.com/ansible/latest/playbook_guide/index.html) in a human-readable data format that is really easy to understand from day one.
+
+## Ansible Installation
+You can use `pipx` to install Ansible in your machine:
+
+```pipx install --include-deps ansible```
+
+Make sure to add the newly created directories in PATH: 
+
+```pipx ensurepath```
+
+For more installation methods, check out the [official installation instructions](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html#installing-ansible).
+
+## Setup Inventory
+An Ansible [inventory](https://docs.ansible.com/ansible/latest/inventory_guide/index.html) is a list of all hosts that will be managed by Ansible. They can be logically grouped by webservers, databases, production, etc. Check out the [official documentation](https://docs.ansible.com/ansible/latest/inventory_guide/intro_inventory.html#inventory-basics-formats-hosts-and-groups
+) to learn more.
```

Comment:
Closing ) on a new line?  Maybe that's OK and won't cause problems...

Reply:
I missed that, thanks!